src/components/form/FormFieldLabel.tsx

```diff
@@ -18,23 +18,43 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 import styled from '@emotion/styled';
-import { type ReactNode, forwardRef } from 'react';
+import { forwardRef } from 'react';
+import { Label } from '../typography';
 
-import { useFormFieldContext } from './FormFieldContext';
+interface Props {
+  children: JSX.Element | string;
```

Comment:
The global `JSX` namespace is deprecated

![image](https://github.com/user-attachments/assets/77edda37-d789-4d01-8454-de8307165a94)

Reply:
Arf it sucks that it's not reported by our tooling